src/importexport/mei/tests/data/hairpin-01.mei

```diff
@@ -122,7 +122,7 @@
                         </layer>
                      </staff>
                      <dynam xml:id="d16couhq" label="f" staff="2" startid="#n1h73eje">f</dynam>
-                     <dir xml:id="dxfp0cj" type="mscore-hairpin mscore-decresc" startid="#n1h73eje" extender="true" endid="#n4ihxav">dim.</dir>
+                     <dir xml:id="dxfp0cj" type="mscore-hairpin mscore-dim" startid="#n1h73eje" extender="true" endid="#n4ihxav">dim.</dir>
```

Comment:
I believe this change is harmless, @rettinghaus, @lpugin ?

Reply:
Shouldn't be much of a problem.

Reply:
You shouldn't test the latest version of the test files with an older version of MuseScore. But this is a warning you may ignore …

Reply:
I see, I tested with 4.4.4. Testing with 4.5 doesn't show that warning